website/src/components/Hero.jsx

```diff
@@ -1,72 +1,43 @@
-        <div className="relative flex items-end lg:col-span-5 lg:row-span-2">
-          <div className="absolute -bottom-12 -top-20 left-0 right-1/2 z-10 rounded-br-6xl bg-blue-600 text-white/10 md:bottom-8 lg:-inset-y-32 lg:left-[-100vw] lg:right-full lg:-mr-40">
-            <GridPattern
-              x="100%"
-              y="100%"
-              patternTransform="translate(112 64)"
-            />
-          </div>
-          <div className="relative z-10 mx-auto flex w-64 rounded-xl bg-slate-600 shadow-xl md:w-80 lg:w-auto">
-            <Image className="w-full" src={coverImage} alt="" priority />
-          </div>
-        </div>
-        <div className="relative px-4 sm:px-6 lg:col-span-7 lg:pb-14 lg:pl-16 lg:pr-0 xl:pl-20">
-          <div className="hidden lg:absolute lg:-top-32 lg:bottom-0 lg:left-[-100vw] lg:right-[-100vw] lg:block lg:bg-slate-100" />
-          <Testimonial />
+    <>
+      <header className=" overflow-hidden py-10">
+        <div className="px-auto text-center">
+          <h1 className="bold text-4xl text-purple-600 sm:text-5xl">
+            Welcome to{' '}
+            <span className=" sm: block bg-gradient-to-r from-purple-600 to-blue-800 bg-clip-text text-4xl text-5xl font-extrabold uppercase tracking-wide text-transparent sm:inline">
+              craftonite
+            </span>
+          </h1>
         </div>
-        <div className="bg-white pt-16 lg:col-span-7 lg:bg-transparent lg:pl-16 lg:pt-0 xl:pl-20">
-          <div className="mx-auto px-4 sm:px-6 md:max-w-2xl md:px-4 lg:px-0">
-            <h1 className="font-display text-5xl font-extrabold text-slate-900 sm:text-6xl">
-              Get lost in the world of icon design.
-            </h1>
-            <p className="mt-4 text-3xl text-slate-600">
-              A book and video course that teaches you how to design your own
-              icons from scratch.
-            </p>
-            <div className="mt-8 flex gap-4">
-              <Button href="#free-chapters" color="blue">
-                Get sample chapter
-              </Button>
-              <Button href="#pricing" variant="outline" color="blue">
-                Buy book
-              </Button>
-            </div>
-          </div>
+      </header>
+      <content className="mx-auto flex w-5/6 flex-col-reverse items-center gap-3 sm:w-2/3 sm:flex-row sm:items-start sm:gap-10">
+        <Image
+          width={300}
+          height={300}
+          src={coverImage}
+          alt="craftonite"
+          priority
+        />
+        <div className="flex flex-col items-start gap-3 sm:gap-5">
+          <h2 className="text-center text-2xl sm:text-start sm:text-3xl ">
+            Here you can find the best videos in the best price for your
+          </h2>
+          <h3 className="text-1xl text-center sm:text-start sm:text-2xl">
```

Comment:
you can move text-center to the div className I think

Reply:
Done